io.github.Nheko-Reborn.Nheko.json

```diff
@@ -0,0 +1,199 @@
+{
+  "id": "io.github.Nheko-Reborn.Nheko",
+  "command": "nheko",
+  "branch": "stable",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
```

Comment:
Update runtime to 5.12

Reply:
Updated to 5.12.  I'll try to rebuild now.